ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/CommandTopic.java

```diff
@@ -86,7 +89,9 @@ public void start() {
       for (ConsumerRecord<byte[], byte[]> record : iterable) {
         try {
           backupRecord(record);
-        } catch (final Exception e) {
+        } catch (final KsqlException e) {
+          log.warn("Error encountered while backing up command.", e);
+        } catch (final KsqlServerException e) {
```

Comment:
we should define a new exception type for this case. this is a very specific error condition, and it should have its own exception class.